<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title></title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 64px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h2 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h3 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h4 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h5 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h6 {
  padding-top: 69px;
  margin-top: -69px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Tom Hill</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="research.html">Research</a>
</li>
<li>
  <a href="data.html">Data</a>
</li>
<li>
  <a href="contact.html">Contact</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="contact.html">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://github.com/tomh1lll">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/TomH1lll">
    <span class="fa fa-twitter fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://scholar.google.com/citations?user=WJPoC9AAAAAJ&amp;hl=en">
    <span class="ai ai-google-scholar ai-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">




</div>


<p><br />
<img src="files/fly_pic2.jpg" style="width:45%; margin-right: 15px" align="left"> I am a postdoctoral researcher at the <a href="https://ku.edu/">University of Kansas</a> in the <a href="http://www.uncklesslab.com/">Unckless lab</a>, in the <a href="https://molecularbiosciences.ku.edu/">Department of Molecular Biosciences</a>. I like to analyze data and test hypotheses concerning host-parasite co-evolution in <em>Drosophila</em> experimentally and with bioinformatics. I also like to play board/video games and think about game mechanics.</p>
<p>I graduated from the <a href="http://www.reading.ac.uk/">University of Reading</a> in the United Kingdom with a BSc in Biological Sciences in 2012, and then from the <a href="https://www.popgen-vienna.at/">Vienna Graduate School of Population Genetics</a> with a PhD in Population Genomics in 2016. For my PhD in the <a href="https://www.flyevolution.net/">Betancourt Lab</a>, I studied the horizontal transfer and fitness costs of transposable elements in <em>Drosophila</em>. Since 2016 I have worked at the University of Kansas, studying co-evolution of <em>Drosophila innubila</em> and their associated DNA virus, DiNV. I also study the evolution of antimicrobial peptides and other immune effector molecules.</p>
<p>My full CV is available <a href="files/Tom%20Hill_CV_2020.pdf">here</a>.</p>
<hr />
<div id="news" class="section level3">
<h3>News</h3>
<p><strong>2020-05-20: New preprint - Rapid divergence of the copulation proteins in the Drosophila dunni group is associated with hybrid post-mating-prezygotic incompatibilities</strong><br> <a href="https://www.biorxiv.org/content/10.1101/2020.05.20.106724v1">Available on BiorXiv</a>, Rob and I submitted a new manuscript where we examine the divergence in the genomes of closely related Caribbean <em>Drosophila</em> species. <br></p>
<p><strong>2020-05-14: New preprints - Recurrent evolution of two competing haplotypes in an insect DNA virus &amp; Selection and demography shape genomic variation in a Sky Island species</strong><br> <a href="https://www.biorxiv.org/content/10.1101/2020.05.14.096024v1">Available on Biorxiv</a>, Rob and I submitted two manuscripts examining the evolution of a virus and its host, we find the virus has two competing types that seem to have been maintained for thousands of years. <br></p>
<p><strong>2019-12-28: Honeymoon in Maui!</strong><br></p>
<p><strong>2019-09-28: Dana and I got married today!</strong><br></p>
<p><strong>2019-08-18: Midwest Population Genetics VI</strong><br> Rob, Jo and I attended this conference in Chicago, presenting our recent work. <br></p>
<p><strong>2019-08-16: Manuscript accepted: A Deep-Learning Approach for Detecting Copy NUmber Variants in Next-Generation Sequencing Data.</strong><br> The paper was accepted and should be published in G3 in the next few weeks.<br></p>
<p><strong>2019-07-25: New preprint - Transposable element dynamics are consistent across the Drosophila phylogeny, despite drastically differing content.</strong><br> <a href="https://www.biorxiv.org/content/10.1101/651059v1">Available on BiorXiv</a>, will be submitted for review as soon as the data is published. Using data I’ve scavenged from Justin and Rob, I’ve compared the TE insertion frequency spectrum of five species, as I’d like to start comparing other species to <em>D. melanogaster</em> more, to identify when <em>D. melanogaster</em> represents other species, and when it’s an exception.<br></p>
<p><strong>2019-05-31: New preprint - A Simple Deep-Learning Approach for Detecting Duplications and Deletions in Next-Generation Sequencing Data.</strong><br> <a href="https://www.biorxiv.org/content/10.1101/657361v1">Available on BiorXiv</a>, Rob and I developed a simple python tool for estimating copy number variation in next-generation sequencing information using machine learning. Soon we’ll be applying it across many species to identify AMP copy number variation.<br></p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
